src/app/dim-ui/destiny-symbols/SymbolsPicker.tsx

```diff
@@ -150,15 +161,15 @@ function SymbolsPickerButton<T extends HTMLTextAreaElement | HTMLInputElement>({
       >
         <span>{symbolsIcon}</span>
       </button>
-      {open && (
-        <Portal>
-          <div ref={tooltipContents} style={{ zIndex: 20 }}>
+      {open &&
+        createPortal(
+          <div ref={tooltipContents}>
             <ClickOutside onClickOutside={() => setOpen(false)}>
               <SymbolsWindow onChooseGlyph={onChooseGlyph} />
             </ClickOutside>
-          </div>
-        </Portal>
-      )}
+          </div>,
+          pressTipRoot.current ?? tempContainer
+        )}
```

Comment:
This is a revert of an earlier change that fixed non-interactivity for the item popup symbols picker, but it broke the dialog symbols picker. The `pointer-events: auto;` change above fixes this in a different way.